File: javaSprint2GladlyGiven/src/main/java/pt/gladlyGivenApi/GladlyGiven/Example/ExampleModel.java
// Author: Tiago Barracha
// ti.barracha@gmail.com

package pt.gladlyGivenApi.GladlyGiven.Example;

import jakarta.persistence.*;

@Entity
public class ExampleModel {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    private String name;

    public ExampleModel(String name) {
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: dotNetSprint2GladyGivenWebAPI/GladyGiven/Models/ExampleModel.cs
// Author: Tiago Barracha
// ti.barracha@gmail.com


using System.ComponentModel.DataAnnotations;

namespace GladyGivenWebAPI.Example
{
    public class ExampleModel
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }
    }
}
